yarn.lock

```diff
@@ -26648,6 +26536,36 @@ webpack@5.44.0, webpack@^5.21.2, webpack@^5.9.0:
     watchpack "^2.2.0"
     webpack-sources "^2.3.0"
 
+"webpack@>=4.0.0 <6.0.0", "webpack@>=4.43.0 <6.0.0":
```

Comment:
can we dedupe all webpack 5s ? there shouldn't be a need to have  more than 2 webpack version installed at this point ( web-components is using 4, the rest is on 5)

Reply:
i rean yarn-dedup. If there is a more fine-grained change to be made, I might need to pull you in to make sure i'm doing it right.